### PR TITLE
fix(collapse): Fixed the bug where collapse did not monitor deeply, causing direct push of tiny-collapse activeNames on PC to fail to respond.

### PR DESCRIPTION
**Pull request by wkif**
# PR

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our [Commit Message Guidelines](https://github.com/opentiny/tiny-vue/blob/dev/CONTRIBUTING.md)
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: pnpm-lock.yaml
Comment (GaoNeng-wWw) [inline pnpm-lock.yaml:1]: 不要提交这个文件

**Review by GaoNeng-wWw — DISMISSED** [review]

File: pnpm-lock.yaml
Comment (wkif) [reply pnpm-lock.yaml:1]: 好嘞

**Review by GaoNeng-wWw — APPROVED** [review]
LGTM

**Review by zzcr — APPROVED** [review]

**Comment by kagol** [comment]
@all-contributors please add @wkif for code.

**Pull request merged**